### ISSUE TITLE
tensorflow_synthetic_benchmark.py vs pytorch_imagenet_resnet50.py

### ISSUE THREAD

**Issue by bapriddy** [uber/horovod#712]
@alsrgv @tgaddair 

I compared the synthetic benchmarks expecting some sort of "reasonable" comparision and was shocked to see the following

pytorch_imagenet_resnet50.py
`Model: resnet50
Batch size: 32
Number of GPUs: 4
Running warmup...
Running benchmark...
Iter #0: 108.3 img/sec per GPU
Iter #1: 107.3 img/sec per GPU
Iter #2: 109.3 img/sec per GPU
Iter #3: 108.7 img/sec per GPU
Iter #4: 107.6 img/sec per GPU
Iter #5: 108.8 img/sec per GPU
Iter #6: 107.6 img/sec per GPU
Iter #7: 108.6 img/sec per GPU
Iter #8: 107.5 img/sec per GPU
Iter #9: 108.3 img/sec per GPU
Img/sec per GPU: 108.2 +-1.2
Total img/sec on 4 GPU(s): 432.8 +-4.9 

vs 

tensorflow_synthetic_benchmark.py

Model: ResNet50
Batch size: 32
Number of GPUs: 4
2018-12-19 14:03:19.293038: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA
2018-12-19 14:03:19.507945: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA
2018-12-19 14:03:19.509022: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA
2018-12-19 14:03:19.530783: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA
Running warmup...
Running benchmark...
Iter #0: 0.6 img/sec per GPU
Iter #1: 0.6 img/sec per GPU 

Am I missing something about benchmarks?  tensorflow_synthetic_benchmark.py seems extremely slow??  Does this use tf.records??

**Comment by alsrgv** [comment]
@bapriddy, is your TensorFlow compiled with GPU support? Usually you'd see GPU initialization messages.

**Comment by bapriddy** [comment]
I'll check.  And reinstall if not.

**Comment by bapriddy** [comment]
Yes, I had the cpu version installed.  Actually I have never used tensorflow before today.  I was just running the code to compare...It look like I'll need to run this job on another system.  Thanks!!
